olivierkes/manuskript
# Fix tab key order, and default window tab for character & plot panes

This code change sets:
- Character pane default first tab of "Basic info" (was Notes)
- Character pane Basic info "Name" as the default first field (was Motivation)
- Plots pane default first tab of "Basic info" (was Resolution steps)

To create this PR I used the following software versions with the _assumption_ that these changes would be forward compatible:
- PyQt 5.5.1
- Qt 5.5.1
- Qt Designer 5.5.1

One thing of note that Qt Designer automatically added is multiple lines similar to:
```
+        MainWindow.setTabOrder(self.tabMain, self.txtGeneralTitle)
+        MainWindow.setTabOrder(self.txtGeneralTitle, self.txtGeneralSubtitle)
+        MainWindow.setTabOrder(self.txtGeneralSubtitle, self.txtGeneralSerie)
...
```
and
```
+<tabstops>
+  <tabstop>tabMain</tabstop>
+  <tabstop>txtGeneralTitle</tabstop>
...
```

To test the default panes:

1. Start Manuskript and create a new project using the **Novel** template
2. Select the **Characters** pane.
   The default tab should be **Basic info** (was **Notes** previously)
3. Select the **Plots** pane.
   The default tab should be **Basic info** (was **Resolution steps** previously).

This Pull Request is ready for review.

Note that I likely won't have a chance to look at this again until Tuesday.
